lib/cc/config/validation/engine_validator.rb

```diff
@@ -42,6 +45,23 @@ def validate_root
           end
           true
         end
+
+        def validate_checks
+          return unless validate_key_type("checks", Hash)
+
+          data.fetch("checks", {}).each do |check_name, check_data|
+            validator = CheckValidator.new(check_data)
+            errors.push(*validator.errors)
+            warnings.push(*validator.warnings)
+          end
+        end
+
+        def validate_exclude_fingerprints
+          validate_key_type("exclude_fingerprints", Array)
+          if data.key?("exclude_fingerprints")
+            warnings << "'exclude_fingerprints' is deprecated, and support may be removed in the future"
```

Comment:
Worth mentioning that they can use issue statuses, instead?

Reply:
Well, if they're primarily CLI users, they can't yet, so I don't want to say that yet.